### PR TITLE
chore(monitoring): Move samples from ruby-docs-samples

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-monitoring/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Install the [Bundler](http://bundler.io/) gem.
+
+1. Install dependencies using:
+
+    `bundle install`
+
+## Run Quickstart
+
+    bundle exec ruby quickstart.rb
```
Comment (quartzmo) [inline google-cloud-monitoring/samples/README.md:43]: Do you want to copy the section for running the tests from another samples README?

File: google-cloud-monitoring/samples/Rakefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rake/testtask"
+
+Rake::TestTask.new "test" do |t|
+  t.test_files = FileList["acceptance/*_test.rb"]
+  t.options = "--junit --junit-filename=sponge_log.xml"
```
Comment (quartzmo) [inline google-cloud-monitoring/samples/Rakefile:19]: I've been removing the `minitest-junit` gem and output in my google-cloud-ruby PRs.

File: google-cloud-monitoring/samples/README.md (hunk 1 above)
Comment (dazuma) [reply google-cloud-monitoring/samples/README.md:43]: done

File: google-cloud-monitoring/samples/Rakefile (hunk 2 above)
Comment (dazuma) [reply google-cloud-monitoring/samples/Rakefile]: done

**Comment by dazuma** [comment]
The `GOOGLE_CLOUD_SAMPLES_TEST` logic is there. I'm using an alternate form that Slava introduced (which relaxes the need for disabling a rubocop linter).

The Rake tasks are also _already_ present. The microgenerator already includes them when it generates the gem-level Rakefile.

**Comment by quartzmo** [comment]
> The Rake tasks are also already present. The microgenerator already includes them when it generates the gem-level Rakefile.

Oh no! I didn't know about the `samples` Rake tasks being generated, and I added them in #6267! I'll create a PR to remove the duplicates.

**Comment by dazuma** [comment]
Oh, I missed that. 😁 Don't worry, autosynth will remove it.

**Comment by quartzmo** [comment]
> The GOOGLE_CLOUD_SAMPLES_TEST logic is there. I'm using an alternate form that Slava introduced (which relaxes the need for disabling a rubocop linter).

I'll use this form from now on.

**Pull request merged**